sentry-android-core/src/test/java/io/sentry/android/core/AndroidTransactionProfilerTest.kt

```diff
@@ -46,6 +49,7 @@ class AndroidTransactionProfilerTest {
         context = ApplicationProvider.getApplicationContext()
         file = context.cacheDir
         AndroidOptionsInitializer.init(fixture.options, createMockContext())
+        Sentry.init(fixture.options)
```

Comment:
Not all the tests require `Sentry.init`, so I'd rather do in a specific test and also call `Sentry.close` after.

Reply:
yeah, it's needed to generate the profiling trace folder. Was missing the `Sentry.close` part 😅